src/include/checkmedia/ui.rb

```diff
@@ -555,6 +552,18 @@ def MainDialog
       ret
     end
 
+    # try to read one byte from the medium to check whether it is readable
+    # @param file [String] file path
+    # @return [Boolean] returns `true` if the file is readable, `false` otherwise
+    def file_readable?(file)
+      File.open(file) do |f|
+        f.read(1)
+      end
+      true
+    rescue StandardError
+      false
```

Comment:
I would at least log content of exception if we get some bug report. So we see how it failed.